src/utils/homerunDistance.ts

```diff
@@ -0,0 +1,26 @@
+// Returns the recorded homerun distance in FEET given the in-game position of the sandbag.
+// The conversion from internal game units to feet and meters is not currently perfectly understood.
```

Comment:
When you say not perfectly understood, does that mean that we don't know the conversion logic at all? Is it not possible currently to make this function accept like a param for the unit to return it in?
e.g. `positionToHomeRunDistance(distance: number, units: 'feet' | meters' = 'feet')`?

Reply:
The math is... not simple. The Stadium community doesn't have a perfectly accurate formula to convert from in-game units to displayed distance. They have a spreadsheet [here](https://docs.google.com/spreadsheets/d/11K1QI_sZi8ePWvejQAG-2uhvBEi7zTdBluscIFPl6Ow/edit#gid=0) that I was pointed to for the math. It works perfectly to most values but some large ones are off.

I've added a conversion from feet to meters.

Reply:
taukhan has since been in contact with me and has supplied me with a much simpler formula for calculating score, which I have updated this file with

the reason their calculations were so complex is because they were keeping track of records in Google Sheets which does not support float32